meta/fdbe07ac.csv

```diff
@@ -76,7 +76,7 @@ Id,FileID,Domain,RepoName,FilePath,LineStart,LineEnd,GroundTruth,WithWords,Value
 42651,4f1e3039,GitHub,fdbe07ac,data/fdbe07ac/src/4f1e3039.yaml,77,77,Template,F,35,41,F,F,Any,,,Secret,2.58,6,F,F,F,Password
 46224,35658be4,GitHub,fdbe07ac,data/fdbe07ac/test/35658be4.go,67,67,T,F,14,78,F,F,Any,,,Unknown,3.91,64,F,F,F,Bitbucket Client Secret
 46225,35658be4,GitHub,fdbe07ac,data/fdbe07ac/test/35658be4.go,68,68,T,F,15,79,F,F,Any,,,Unknown,3.87,64,F,F,F,Bitbucket Client Secret
-46228,84760b8a,GitHub,fdbe07ac,data/fdbe07ac/test/84760b8a.go,42,42,T,T,7,15,F,F,Any,,,Unknown,2.75,8,F,F,F,Password
+46228,84760b8a,GitHub,fdbe07ac,data/fdbe07ac/test/84760b8a.go,42,42,T,T,7,15,F,F,Any,,,Unknown,2.75,8,F,F,F,Other
```

Comment:
And this is password too. Previous markup is correct.

Reply:
Not supported yet. Perhaps it will be with different  name